docs-2.0/nebula-exchange/use-exchange/ex-ug-import-from-csv.md

```diff
@@ -190,56 +191,84 @@ After Exchange is compiled, copy the conf file `target/classes/application.conf`
       # The delimiter specified. The default value is comma.
       separator: ","
 
-      # If the CSV file have header, set the header to true.
-      # If the CSV file does not have header, set the header to false. The default value is false.
+      # If the CSV file has a header, set the header to true.
+      # If the CSV file does not have a header, set the header to false. The default value is false.
       header: false
 
-      # Number of pieces of data written to Nebula Graph in a single batch.
+      # The number of data written to Nebula Graph in a single batch.
       batch: 256
 
-      # Number of Spark partitions
+      # The number of Spark partitions.
       partition: 32
     }
 
-    # Set Tag Team information.
+    # Set the information about the Tag Team.
     {
+      # Specify the Tag name defined in Nebula Graph.
       name: team
       type: {
+        # Specify the data source file format to CSV.
         source: csv
+
+        # Specify how to import the data into Nebula Graph: Client or SST.
         sink: client
       }
+
+      # Specify the path to the CSV file.
+      # If the file is stored in HDFS, use double quotation marks to enclose the file path, starting with hdfs://. For example: "hdfs://ip:port/xx/xx".
+      # If the file is stored locally, use double quotation marks to enclose the file path, starting with file://. For example: "file:///tmp/xx.csv".
       path: "hdfs://192.168.*.*:9000/data/vertex_team.csv"
+
+      # If the CSV file does not have a header, use [_c0, _c1, _c2, ..., _cn] to represent its header and indicate the columns as the source of the property values.
+      # If the CSV file has headers, use the actual column names.
       fields: [_c1]
+
+      # Specify the column names in the player table in fields, and their corresponding values are specified as properties in the Nebula Graph.
+      # The sequence of fields and nebula.fields must correspond to each other.
       nebula.fields: [name]
+
+      # Specify a column of data in the table as the source of vertex VID in the Nebula Graph.
```

Comment:
VID stands for vertex ID. Here `vertex ID` would be enough. 
And `the source of vertex ID` does not make sense. Do you mean the ID of the source vertex?